src/Microsoft.AspNet.Server.Kestrel/KestrelEngine.cs

```diff
@@ -1,26 +1,31 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using Microsoft.AspNet.Server.Kestrel.Networking;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 using Microsoft.AspNet.Server.Kestrel.Http;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Logging.Console;
+using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Threading.Tasks;
 
 namespace Microsoft.AspNet.Server.Kestrel
 {
     public class KestrelEngine : IDisposable
     {
 
-        public KestrelEngine(ILibraryManager libraryManager)
+        public KestrelEngine(ILibraryManager libraryManager, ILoggerFactory loggerFactory)
         {
             Threads = new List<KestrelThread>();
             Listeners = new List<Listener>();
             Memory = new MemoryPool();
             Libuv = new Libuv();
 
+            loggerFactory.AddConsole();
```

Comment:
Yeah, Kestrel should not call `AddConsole()`. WebListener deals with this by using Logger.WriteInformation and then we put `AddConsole()` in the `Startup` samples.

I'd recommend against doing Console.WriteLine's, together with or in place of Logger. You want all the information to go to the logger where you can configure the output more directly (to console, disk, remote, etc.). Having this one piece of information take a different rout is just going to make it get lost.